sources/yaml/src/test/resources/example-collections.yml

```diff
@@ -0,0 +1,11 @@
+application:
```

Comment:
do you support also collection of native types?
```
application:
  elements:
    - foo
    - bar
...
```

Reply:
Not yet. That is something I need to figure out. The  issue is that any `Collection` type with a type supported natively by MP Config with a `Converter` will have priority and that is not what we want to do here.

Reply:
I've pushed some new code that should provide the foundation to support this as well. There are still some details I'll need to figure out about the YAMLSource.